app/assets/javascripts/preferences/PreferencesView.tsx

```diff
@@ -37,19 +32,27 @@ const PaneSelector: FunctionComponent<{
 });
 
 const PreferencesCanvas: FunctionComponent<{
-  preferences: Preferences;
+  preferences: PreferencesMenu;
 }> = observer(({ preferences: prefs }) => (
   <div className="flex flex-row flex-grow min-h-0 justify-between">
-    <PreferencesMenu preferences={prefs}></PreferencesMenu>
+    <PreferencesMenuView menu={prefs}></PreferencesMenuView>
     <PaneSelector prefs={prefs} />
   </div>
 ));
 
-const PreferencesView: FunctionComponent<PreferencesViewProps> = observer(
+const blockPropagation = (e: MouseEvent) => {
+  e.stopPropagation();
+};
+
+const PreferencesView: FunctionComponent<{ close: () => void }> = observer(
   ({ close }) => {
-    const prefs = new Preferences();
+    const prefs = new PreferencesMenu();
+
     return (
-      <div className="sn-full-screen flex flex-col bg-contrast z-index-preferences">
+      <div
+        onClick={blockPropagation}
```

Comment:
Could you explain a bit why we need this onClick since it's only stopping propagation?

Reply:
We don't, I missed this, it's a remnant of a bug I was fixing. Removed in 311f50d00cef43129e5fa08819e9987a3625ee87 .